compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAssignPacketIds.cpp

```diff
@@ -41,45 +42,50 @@ void AMDAIEAssignPacketIdsPass::runOnOperation() {
   auto ui8ty =
       IntegerType::get(rewriter.getContext(), 8, IntegerType::Unsigned);
 
+  // Collect all packet flow operations and categorize them into control and
+  // normal flows. Control packet flows will be prioritized for packet ID
+  // assignment.
+  SmallVector<AMDAIE::FlowOp> ctrlPktFlowOps;
+  SmallVector<AMDAIE::FlowOp> normalPktFlowOps;
```

Comment:
I suggest`dataPktFlowOps` as I think it's more descriptive.
```suggestion
  SmallVector<AMDAIE::FlowOp> dataPktFlowOps;
```